ReactWindows/ReactNative.Net46/Modules/Dialog/DialogModule.cs

```diff
@@ -5,16 +5,19 @@
 
 using Newtonsoft.Json.Linq;
 using ReactNative.Bridge;
+using ReactNative.Bridge.Queue;
 using ReactNative.Json;
 using System.Windows;
 
 namespace ReactNative.Modules.Dialog
 {
     class DialogModule : ReactContextNativeModuleBase
     {
+
         public DialogModule(ReactContext reactContext)
-            : base(reactContext)
+            : base(reactContext, new DispatcherActionQueue(reactContext.NativeModuleCallExceptionHandler))
```

Comment:
@rozele you had me remove the NativeModuleCallExceptionHandler from IReactContext in #1768. does this seemingly appropriate, common usage mean we should add it back?

Reply:
@rozele While launching app in ReleaseBundle configuration app is getting below exception. 
Exception thrown: 'System.ArgumentNullException' in ReactNative.Net46.dll at ReactContextNativeModuleBase.cs file Line 38.

But when ran in Debug, Release, DebugBundle configuration - App launched successfully without any issues.

Also, in ReleaseBundlewhen if I enabled DEBUG constant (Properties->Build) - App launched successfully without any issues.

I think this is some timing related issue, in Release mode reactContext is not initialized correctly.

Can you please check and let me know your inputs, if I missing something.

Reply:
@piyush965 - I pushed a change that should fix the `ArgumentNullException` in ReleaseBundle (`reactContext.NativeModuleCallExceptionHandler` -> `reactContext.HandleException`), can you try it out?

Reply:
@rozele  I tried the updated changes and it worked now correctly in ReleaseBundle. Thanks for quick response. Save my day!!